src/ts/background/handlers/Extension.ts

```diff
@@ -69,18 +69,18 @@ export default class Extension {
   private signingApprove ({ id, password }: MessageExtrinsicSignApprove): boolean {
     const queued = this.state.getSignRequest(id)
     assert(queued, 'Unable to find request')
-    const { request: { address, genesisHash, method, nonce }, resolve, reject } = queued
-    if (!keyringVault.accountExists(address)) {
+    const { request, resolve, reject } = queued
+    if (!keyringVault.accountExists(request.address)) {
       reject(new Error('Unable to find account'))
       return false
     }
-    const payload = new SignaturePayloadRaw({ blockHash: genesisHash, method, nonce })
+    const payload = createType('ExtrinsicPayload', request, { version: request.version })
```

Comment:
move it to a const or we get it from polkadot const ?

Reply:
For the same reason keeping `MessageExtrinsicSign` instead of replacing it with SignerPayload, I keep this piece of code the same as Jaco's without refactoring to make it easier to sync his future changes.